apps/ledger-live-desktop/package.json

```diff
@@ -57,6 +57,7 @@
   "dependencies": {
     "@braze/web-sdk": "4.10.2",
     "@ledgerhq/coin-bitcoin": "workspace:^",
+    "@ledgerhq/coin-canton": "workspace:^",
```

Comment:
Why ?

Reply:
to be able to import types `import { OnboardStatus } from "@ledgerhq/coin-canton/types";`